### PR TITLE
[Symfony Bridge][Extension] EventDispatcherExtension - compatible with new Extension API

### PR DESCRIPTION
**Pull request by mkurzeja**
PR #294 updated to the new Extension API

### CODE REVIEW

File: src/Payum/Core/Bridge/Symfony/Event/ExecuteEvent.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+namespace Payum\Core\Bridge\Symfony\Event;
+
+use Payum\Core\Extension\Context;
+use Symfony\Component\EventDispatcher\Event;
+
+
+/**
+ * Class ExecuteEvent
+ * 
+ * @package Payum\Core\Bridge\Symfony\Event
```
Comment (makasim) [inline src/Payum/Core/Bridge/Symfony/Event/ExecuteEvent.php]: remove this docblock

File: src/Payum/Core/Bridge/Symfony/Event/ExecuteEvent.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+namespace Payum\Core\Bridge\Symfony\Event;
```
Comment (makasim) [inline src/Payum/Core/Bridge/Symfony/Event/ExecuteEvent.php]: remove extra spaces

File: src/Payum/Core/Bridge/Symfony/PayumEvents.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Payum\Core\Bridge\Symfony;
+
+/**
+ * @author Abdellatif Ait boudad <a.aitboudad@gmail.com>
+ */
+final class PayumEvents
+{
+    const PAYMENT_PRE_EXECUTE = 'payum.payment.pre_execute';
```
Comment (makasim) [inline src/Payum/Core/Bridge/Symfony/PayumEvents.php]: GATEWAY_XXX events

Comment (makasim) [inline src/Payum/Core/Bridge/Symfony/PayumEvents.php]: const GATEWAY_PRE_EXECUTE = 'payum.gateway.pre_execute';

File: src/Payum/Core/Bridge/Symfony/Extension/EventDispatcherExtension.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+use Payum\Core\Bridge\Symfony\PayumEvents;
+
+class EventDispatcherExtension implements ExtensionInterface
+{
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
+
+    public function __construct(EventDispatcherInterface $dispatcher)
```
Comment (makasim) [inline src/Payum/Core/Bridge/Symfony/Extension/EventDispatcherExtension.php]: add docblock

**Comment by makasim** [comment]
Looks good to me

**Comment by mkurzeja** [comment]
Sorry for the cs issues, fixed now.

**Comment by makasim** [comment]
Thanks!

**Pull request merged**